index.bs

```diff
@@ -1272,10 +1264,9 @@ run the following steps:
     as long as it is in the past.
 
 1. Let |value| be the empty string.
-1. Let |secure| be true.
```

Comment:
`|secure|` is still passed below as an argument (line 1288). It needs to be removed from the list of passed arguments.